entrypoint.sh

```diff
@@ -49,7 +49,7 @@ fi
 if [ "${USE_AUTO_FOCUS}" = "1" ]; then
     ROS_FLAGS="${ROS_FLAGS} use_auto_focus:=true"
 fi
-
+/etc/init.d/udev start
```

Comment:
if an entrypoint starts udev inside of a container (which is highly unusual) the *why* needs to be commented

Reply:
I didn't find explicit mentions of this online but I asked ChatGPT and it gave the exact answer that summarizes my intuition:

![image](https://github.com/tiiuae/depthai_ctrl/assets/630151/1bf76c07-10ae-4e57-aec8-c30e64d65305)